scripts/auto_release/main.py

```diff
@@ -403,6 +410,8 @@ def check_ci_file(self):
 
     def check_file(self):
         self.check_pprint_name()
+        self.check_sdk_readme_and_setup()
```

Comment:
I think `self.check_sdk_readme_and_setup()` shall be put before `self.check_pprint_name()`